Makefile

```diff
@@ -30,7 +30,9 @@ php = docker-compose exec -T --user $(CUID):$(CGID) php ${1}
 php-0 = docker-compose exec -T php ${1}
 
 ## Full site install from the scratch
-all: | provision composer si localize hooksymlink info
+all: | provision composer front si localize hooksymlink info
+## Full site install from the scratch without front task(it managed in .gitlab-ci).
+all_ci: | provision composer si localize hooksymlink info
```

Comment:
Not sure at all if CI job `prepare:nodejs` should exist : What is being done in this job should be done in `deploy:review` instead (or it's template). Done this way, all_ci has no reason to exists

Reply:
Follow up issue created in contrib board